xcp_d/interfaces/prepostcleaning.py

```diff
@@ -45,9 +48,11 @@ class _RemoveTROutputSpec(TraitedSpec):
                                 mandatory=True,
                                 desc="bold or cifti with volumes dropped")
 
-    custom_confounds_dropped = File(exists=False,
-                                    mandatory=False,
-                                    desc="custom_confounds_tsv dropped")
+    custom_confounds_dropped = traits.Either(traits.Undefined,
+                                     File,
+                                     desc="Custom confounds file with volumes dropped",
+                                     exists=False,
+                                     mandatory=False)
```

Comment:
```suggestion
    custom_confounds_dropped = traits.Either(
        traits.Undefined,
        File(exists=True),
        desc="Custom confounds file with volumes dropped",
        mandatory=False,
    )
```

Reply:
Since the file must exist, if the output is a File, this probably works better. Plus it should flag any non-file strings.